src/param.js

```diff
@@ -217,6 +217,7 @@ module.exports.List = (...args) => new List(...args);
 class FieldsParam extends Str {
   constructor(name, paths, ...args) {
```

Comment:
I was thinking the option to prune or not to prune should come in here. That way we can have multiple fields parameter, but only one with "autoPrune = true"

Reply:
I was thinking the same thing. Having it globally isn't great because then you can't specify per endpoint. I'm going to change this